test-in-renode/run_renode_test.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+set -e
+if ! ../__tests__/check_renode_install.sh;
+then
+    RENODE_DIR=$(mktemp -d)
+    echo "RENODE_DIR=$RENODE_DIR" >> $GITHUB_ENV
+    wget https://dl.antmicro.com/projects/renode/builds/renode-$RENODE_VERSION.linux-portable.tar.gz || echo "Wrong Renode version: $RENODE_VERSION"
```

Comment:
Using `||` results in hiding the error, so the script will move forward.

Reply:
Plus I would reprhase the message saying that there was an error when downloading the package - it might be that the problem is not related to the wrong version but some temporary network problems.

Reply:
I mean mentioning that perhaps the provided version is wrong should stay as well.

Reply:
also please make the wget more quiet, we don't need this massive output

Reply:
In general I see that our output is veeeery polluted. We should use folding to hide out all non-important stuff, like wget, pip, tar etc